.github/workflows/docker-main.yaml

```diff
@@ -18,6 +18,8 @@ jobs:
       matrix:
         image:
           - name: app-engine
+            context: '.'
+            file: app-engine
```

Comment:
I suggest instead of `file` name it `folder` or `directory` to avoid any misunderstanding